src/api/spec/controllers/webui/package_controller_spec.rb

```diff
@@ -1,5 +1,5 @@
-require 'rails_helper'
 require "webmock/rspec"
+require 'rails_helper'
```

Comment:
I didn't... was as a result of a rebase... but I doesn't change anything at all.. right?